modules/flowable-cmmn-rest/src/main/java/org/flowable/cmmn/rest/service/api/runtime/variable/VariableInstanceResponse.java

```diff
@@ -26,6 +26,7 @@ public class VariableInstanceResponse {
     protected String caseInstanceId;
     protected String caseInstanceUrl;
     protected String taskId;
+    protected String subScopeId;
```

Comment:
Let's name this `planItemInstanceId`, similar to how we are not using `scopeId`, but `caseInstanceId`